confluent/docker_utils/cub.py

```diff
@@ -94,10 +94,13 @@ def __request(host, port, secure, ignore_cert, username='', password='', path=''
     return requests.get(url, verify = not ignore_cert, auth = auth)
 
 def log4j_config_file():
+    def default_config = "/etc/cp-base-new/log4j.properties"
```

Comment:
I think you could reduce duplication like so:
```
    default_config = "cp-base-new"
    component = os.environ.get("COMPONENT") if os.environ.get("COMPONENT") else default_config
    component_config = "/etc/" + component + "/log4j.properties"
    # check component_config exists, else default to cp-base-new
    if os.path.exists(component_config):
        return component_config
    return default_config
```

Reply:
I ended up moving the default to a constant, does this look slightly better now? 👀